dcicutils/env_utils.py

```diff
@@ -369,23 +470,67 @@ def infer_repo_from_env(envname):
         return None
 
 
+# TODO: Figure out if these two actually designate different hosts or if we could make CGAP prettier. -kmp 4-Oct-2021
+CGAP_FOURSIGHT_URL_PREFIX = "https://u9feld4va7.execute-api.us-east-1.amazonaws.com/api/view/"
+FF_FOURSIGHT_URL_PREFIX = "https://foursight.4dnucleome.org/view/"
+
+
+def infer_foursight_url_from_env(request, envname):
+    token = infer_foursight_from_env(request, envname)
+    if token is not None:
+        prefix = CGAP_FOURSIGHT_URL_PREFIX if is_cgap_env(envname) else FF_FOURSIGHT_URL_PREFIX
+        return prefix + token
+    else:
+        return None
```

Comment:
Right. That comes later in [utils PR #149](https://github.com/4dn-dcic/utils/pull/149), which needs to be extended with support for these functions once we're sure this is the set. Implementing those is easier than figuring out what will span the set of functionalities we need. It also won't involve detailed changes to the other repos. It should mostly just slide right in with a version upgrade for `dcicutils`. I'll bump the major version pro-forma on that change, just because it's a big one, but I think it will be a compatible release.